$MirOS: ports/games/bsdgames/patches/patch-rogue_USD_doc_Makefile,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- rogue/USD.doc/Makefile.orig	Thu Jan  1 00:00:00 1970
+++ rogue/USD.doc/Makefile	Sat Nov 19 22:53:35 2005
@@ -5,7 +5,11 @@ DIR=	usd/30.rogue
 SRCS=	rogue.me
 MACROS=	-me
 
-paper.${PRINTER}: ${SRCS}
-	${TBL} ${SRCS} | ${ROFF} > ${.TARGET}
-
-.include <bsd.doc.mk>
+.ifdef TRUEPREFIX
+# MirPorts  
+install:
+	${INSTALL} ${INSTALL_COPY} -o ${DOCOWN} -g ${DOCGRP} -m ${DOCMODE} \
+	    Makefile ${SRCS} ${DESTDIR}${DOCDIR}/${DIR}/
+.else
+.  include <bsd.doc.mk>
+.endif
